doc/voq/bgp_voq_chassis.md

```diff
@@ -0,0 +1,164 @@
+# BGP Setup for VoQ Chassis
+# High Level Design Document
+
+### Rev 1.0
+
+# Table of Contents
+* [List of Figures](#list-of-figures)
+* [Revision](#revision)
+* [About this Manual](#about-this-manual)
+* [Scope](#scope)
+* [1 Requirements](#1-requirements)
+* [2 Design](#2-design)
+  * [2.1 BGP Setup](#21-bgp-setup)
+  * [2.2 iBGP Configuration](#22-ibgp-configuration)
+  * [2.3 FRR Changes](#23-frr-changes)
+  * [2.4 CONFIG_DB Changes](#24-config_db-changes)
+  * [2.5 FRR Configuration Templates](#25-frr-configuration-templates)
+  * [2.6 Mingraph Changes](#26-minigraph-changes)
+* [3 Testing](#3-testing)
+
+# Revision
+| Rev |     Date    |       Author       | Change Description |
+|:---:|:-----------:|:------------------:|--------------------|
+| 1 | Sep-10 2020 | Joanne Mikkelson (Arista Networks) | Initial Version |
+
+# About this Manual
+
+This document describes how BGP will be configured on a VoQ chassis.
+
+# Scope
+
+The configuration described here is intended to ensure that routes programmed
+for BGP-learned prefixes are the same for each FSI in a VoQ chassis.
+
+This document describes:
+- BGP configuration changes
+- Changes to Minigraph and CONFIG_DB
+- FRR configuration templates
+
+# 1 Requirements
+
+Traffic ingressing on any FSI should be routed the same, regardless of which
```

Comment:
Clarify that forwarding behavior can be different, but the nexthop set for the ECMP will be same

Reply:
Clarify that this is substantially different from an internal-ethernet-clos based chassis because the forwarding decision is made once at the ingress in voq chassis and several times in an ethernet based chassis.